packages/backend/package.json

```diff
@@ -35,7 +35,7 @@
     "publish:local": "npx yalc push --replace --sig",
     "build:lib": "tsup --env.NODE_ENV production",
     "build:tests": "tsc -p tsconfig.test.json",
-    "build:runtime": "rsync -r --include '*/' --include '*.js' --include '*.mjs' --exclude='*' src/runtime dist",
+    "build:runtime": "cpy 'src/runtime/**/*.{mjs,js,cjs}' dist/runtime",
```

Comment:
I was using https://github.com/nektos/act to test this workflow locally and `rsync` isn't installed in every linux distro. So I switched to a package that'll work everywhere (and is easier to grok)